platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/dispatch/DispatchBuilder.java

```diff
@@ -161,9 +163,8 @@ DispatchBuilder registerObserver(
                     final String comment) {
 
         throwIfImmutable("observer can only be registered while this object is mutable");
-        throwArgNull(triggerClass, "triggerClass");
-        throwArgNull(triggerClass, "dispatchType");
-        throwArgNull(observer, "observer");
+        Objects.requireNonNull(triggerClass, "triggerClass must not be null");
+        Objects.requireNonNull(observer, "observer must not be null");
```

Comment:
Removed the check 
`throwArgNull(triggerClass, "dispatchType");`
it seamed an error.